docs/user-docs.md

```diff
@@ -515,19 +515,19 @@ more][13].
 
     Or, for a Python job
 
-        $ dcos spark run --submit-args="http://external.website/mysparkapp.py 30"
+        $ dcos spark run --submit-args="http://external.website/mysparkapp.py --py-files mydependency.py 30"
```

Comment:
Does --py-files need to go before the app (mysparkapp.py)? I'm not sure if this will be parsed correctly. 

Otherwise, LGTM.